src/styles/_class-overrides.scss

```diff
@@ -347,6 +347,10 @@
     padding: 0;
   }
 
+  #feedback-widget{
```

Comment:
I know this is fine but Justin said we should stick to using classes only. Do you think instead we should add a class name and use it here?

Reply:
Yeah I just quickly wanted to update the styling before the show and tell, and there was already an id. I updated it to include a class too